uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -86,6 +87,49 @@ public static function findPublicHolidayLeaveRequest($contactID, PublicHoliday $
   }
 
   /**
+   * Returns a list of all Absence Types, together with its total balance change.
+   * That is, the sum of all the Leave Balance Changes for Leave Requests of that
+   * Absence Type, for the given $contactID during the given $periodID.
+   *
+   * Balance Changes for Public Holiday Leave Requests won't be considered,
+   * except when $publicHolidays is true. In that case, on the balance changes
+   * for that type of request will be considered.
+   *
+   * @param int $contactID
+   * @param int $periodID
+   * @param array $leaveRequestStatus
+   *   And array of values from the Leave Request Status OptionGroup
+   * @param bool $publicHolidays
+   *   When true, will get the balance change only for the Public Holiday Leave Requests
+   *
+   * @return array
+   */
+  public static function getBalanceChangeByAbsenceType(
+    $contactID,
+    $periodID,
+    $leaveRequestStatus = [],
+    $publicHolidays = false
+  ) {
+    $periodEntitlements = LeavePeriodEntitlement::getPeriodEntitlementsForContact($contactID, $periodID);
+
+    $results = [];
+    $excludePublicHolidays = $publicHolidays ? false : true;
```

Comment:
We can replace this with `$excludePublicHolidays = !($publicHolidays);`
Or what do you think?